src/PortingAssistant.Client.Porting/PortingProjectFile/PortingProjectFileHandler.cs

```diff
@@ -35,31 +35,36 @@ public PortingProjectFileHandler(ILogger<PortingProjectFileHandler> logger)
         /// <returns>A PortingProjectFileResult object, representing the result of the porting operation</returns>
         /// 
         public List<PortingResult> ApplyProjectChanges(
-            List<string> projectPaths, string solutionPath, string targetFramework,
+            List<ProjectDetails> projects, string solutionPath, string targetFramework,
             Dictionary<string, Tuple<string, string>> upgradeVersions)
         {
-            _logger.LogInformation("Applying porting changes to {0}", projectPaths);
+            _logger.LogInformation("Applying porting changes to {0}", solutionPath);

```

Comment:
What would that display if we're running this on one project?

Reply:
I intended to avoid the large content of projectDetail, but it looks like it's better to use the old projectPath to match the changes